pkgs/by-name/vs/vscode-runner/update.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p yq ripgrep common-updater-scripts dart
+
+set -xeu -o pipefail
+
+PACKAGE_DIR="$(realpath "$(dirname "$0")")"
+cd "$PACKAGE_DIR/.."
+while ! test -f flake.nix; do cd ..; done
+NIXPKGS_DIR="$PWD"
+
+# Get latest version number from GitHub
+version="$(
+  list-git-tags --url=https://github.com/Merrit/vscode-runner |
+    rg '^v(.*)' -r '$1' |
```

Comment:
for documentation and scripts, to improve readability and be more explicit, i recommend the long options when available

for example, in this case

```suggestion
    rg '^v(.*)' --replace '$1' |
```